src/main/java/com/performance/demo/performance/dao/Cpu.java

```diff
@@ -11,9 +11,17 @@ public class Cpu extends BaseMeasurement {
     @Column(name = "value")
     private Double value;
 
-    public Cpu(Double value, Instant time, String flowName, String userName) {
+    @Column(tag = true, name = "action_name")
```

Comment:
I think we can avoid duplicates of actionName and elementName of different measurements by adding these fields to BaseMeasurement and creating additional constructor